.github/workflows/backend.yml

```diff
@@ -56,7 +56,28 @@ jobs:
     - name: Tests
       shell: bash
       run: |
-        bash scripts/ci.sh
+        bash scripts/test.sh
+  client-lint:
```

Comment:
Thanks for the PR @akx . I think `needs` is the right approach here: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds

That way the tests don't run if lint fails